andi/andi.py

```diff
@@ -75,8 +93,9 @@ def dependencies(self) -> List[Step]:
         return self[:-1]
 
     @property
-    def final_arguments(self) -> Dict[str, Callable]:
+    def final_kwargs_spec(self) -> KwargsSpec:
         """
+        TODO: should we remove this function at all?
```

Comment:
Yeah, it looks fine to remove it, if we're not using it anywhere.

Reply:
We use in the tests... but we can replate it very easily. So I'll remove it.